public/samples/Automation/tutorials/VRFSubscriptionBalanceMonitor.sol

```diff
@@ -0,0 +1,336 @@
+// SPDX-License-Identifier: MIT
```

Comment:
We can probably leave this code sample at the source and just copy in snippets for the code analysis.

Reply:
Disregard this comment. We'll need to host it locally similar to ETH Balance Monitor.